src/wallet/rpcnames.cpp

```diff
@@ -616,7 +627,14 @@ name_update ()
                           " on this name");
 
     if (pendingOps > 0)
-      outp = mempool.lastNameOutput (name);
+      {
+        outp = mempool.lastNameOutput (name);
+        if (isDefaultVal)
+          {
+            value = CNameScript(mempool.mapTx.find(outp.hash)->GetTx()
+            .vout[outp.n].scriptPubKey).getOpValue();
```

Comment:
This looks confusing in terms of indentation (as it is not clear immediately that it is a continuation of the line, rather than a new line by itself).  I'm not sure how to best format it, perhaps I would define a `const auto&` for the tx or something like that.

Also, I think the usage of `find` here is not ideal (since you are not checking it against being equal to `end`).  Instead, I would suggest to use `at` for the lookup.  (Or use `find` but also `assert` that it is not `end` afterwards.)